splitio/sync/synchronizer.py

```diff
@@ -233,7 +233,13 @@ def synchronize_splits(self, till):
         """
         _LOGGER.debug('Starting splits synchronization')
         try:
-            self._split_synchronizers.split_sync.synchronize_splits(till)
+            segment_list = self._split_synchronizers.split_sync.synchronize_splits(till, self._split_synchronizers.segment_sync)
```

Comment:
```
new_segments = [
    segment if not segment_sync.segment_exist_in_storage(segment)
    for segment in self._split_synchronizers.split_sync.synchronize_splits(till, self._split_synchronizers.segment_sync)
]

Reply:
This was similar to the old logic before Matti asked to move it to sync/split.py

Reply:
We will move with this recommendation.